syncapi/storage/sqlite3/current_room_state_table.go

```diff
@@ -51,7 +51,7 @@ CREATE UNIQUE INDEX IF NOT EXISTS syncapi_event_id_idx ON syncapi_current_room_s
 const upsertRoomStateSQL = "" +
 	"INSERT INTO syncapi_current_room_state (room_id, event_id, type, sender, contains_url, state_key, headered_event_json, membership, added_at)" +
 	" VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)" +
-	" ON CONFLICT (event_id, room_id, type, sender, contains_url)" +
+	" ON CONFLICT (room_id, type, state_key)" +
```

Comment:
Why was this changed?

Reply:
Because it's wrong and looks like it was copy-pasta'd from somewhere when we did the first round of SQLite-ing. The Postgres equivalent is `CONSTRAINT syncapi_room_state_unique UNIQUE (room_id, type, state_key)`.